bc_obps/registration/migrations/0003_alter_historicaloperator_bc_corporate_registry_number_and_more.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 4.2.8 on 2023-12-12 19:17
```

Comment:
I assume I must have missed something as there are now three migrations files. I was under the impression that we were deleting the `001` and regenerating it until we release to prod?

Reply:
Yup, I'll delete and regenerate. Just had to check in teams that that was still the plan now that we have history tables